docs/src/design/chiplets/hasher.md

```diff
@@ -107,7 +107,7 @@ Computing a 2-to-1 hash involves the following steps:
 
 1. Initialize hasher state with $8$ field elements, setting the second capacity element to $domain$ if the domain is provided (as in the case of [control block hashing](../programs.md#program-hash-computation)) or else $0$, and the remaining capacity elements to $0$.
 2. Apply Rescue Prime Optimized permutation.
-3. Return elements ${4, ..., 7}$ of the hasher state as output.
+3. Return elements $[4..8)$ of the hasher state as output.
```

Comment:
I thought we decided to do it as $[4, 8)$. Or are there issues with this?

Reply:
Ah, my bad, I got it mixed up, I'll fix it now